openslides/urls.py

```diff
@@ -31,7 +31,7 @@
     (r'^agenda/', include('openslides.agenda.urls')),
     (r'^motion/', include('openslides.motion.urls')),
     (r'^assignment/', include('openslides.assignment.urls')),
-    (r'^user/', include('openslides.users.urls')),
+    (r'^users/', include('openslides.users.urls')),
```

Comment:
Ok, I changed line 16 to "users.".
But here is still a problem with the django urls. I can not run the url "users/print" to generate PDF. For this URL I get a 404 for "/rest/users/user/print"... Any idea?